src/app/(app)/members/components/MemberFormBuilder.tsx

```diff
@@ -136,11 +138,20 @@ export const MemberFormBuilder = ({ member }: MemberFormBuilder) => {
     mode: "all",
   });
 
+
+
+
+
+
   const {
     formState: { errors },
   } = form;
 
+  const dateformatter = (date: Date) => {
```

Comment:
I'm confused , why you using this function , even though you are returing value as it is.

Reply:
fixed